OSPSuite.Presentation.Importer/Views/ImportConfirmationView.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Presentation.Importer.Presenters;
+using OSPSuite.Assets;
+using OSPSuite.UI.Controls;
+
+namespace OSPSuite.Presentation.Importer.Views
+{
+   public partial class ImportConfirmationView : BaseUserControl, IImportConfirmationView
+   {
+      private IImportConfirmationPresenter _presenter;
+      public ImportConfirmationView()
+      {
+         InitializeComponent();
+         layoutControlItem1.Text = Captions.Importer.NamingPattern;
```

Comment:
no default naming for user control

Reply:
all controls named in a meaningful way now